src/cheat_sheet.py

```diff
@@ -93,33 +95,48 @@ def draw(self, canvas):
         self.y = get_y(canvas)
         self.w = 0
 
-        simple_actions = get_list(
-            "simple_action",
+        all_actions = {}
+        for name in ACTION_LIST_NAMES:
+            all_actions.update(get_raw_list(name))
+
+        multiple_target_action_names = [
+            "replaceWithTarget",
+            "moveToTarget",
+            "swapTargets",
+            "applyFormatter",
+        ]
+        simple_actions = {
+            key: value
+            for key, value in all_actions.items()
+            if value not in multiple_target_action_names
+        }
+        complex_actions = {
+            value: key
+            for key, value in all_actions.items()
+            if value in multiple_target_action_names
+        }
+
+        swap_connective = list(get_raw_list("swap_connective").keys())[0]
+        source_destination_connective = list(
+            get_raw_list("source_destination_connective").keys()
+        )[0]
+
+        make_dict_readable(
+            simple_actions,
             {
-                "call": "Call T on S",
-                "define": "Reveal definition",
-                "drink cell": "Edit new cell above",
-                "pour cell": "Edit new cell below",
-                "reference": "Show references",
+                "callAsFunction": "Call T on S",
+                "wrapWithPairedDelimiter": '"round" wrap T',
             },
         )
-
-        move_bring = {}
-        for action, desc in get_list("move_bring_action").items():
-            if desc == "Bring":
-                move_bring[f"{action} T1 to T2"] = "Replace T2 with T1"
-                move_bring[f"{action} T"] = "Replace S with T"
-            if desc == "Move":
-                move_bring[f"{action} T1 to T2"] = "Move T1 to T2"
-                move_bring[f"{action} T"] = "Move T to S"
-
         all_actions = {
             **simple_actions,
-            **move_bring,
-            "swap T1 with T2": "Swap T1 with T2",
-            "swap with T": "Swap S with T",
-            "wrap": '"round" wrap T',
-            "format * at T": "Reformat T as *",
+            f"{complex_actions['replaceWithTarget']} T1 {source_destination_connective} T2": "Replace T2 with T1",
```

Comment:
@AndreasArvidsson works fine as an f-string for me.  Were you trying to use double quotes here?  Can't use the same type of quote as you use to delimit the string, so maybe that's what you were hitting

Reply:
Ah yeah probably the reason. I'm to used to js/ts.